routes/userDeviceInfo.js

```diff
@@ -0,0 +1,8 @@
+const express = require("express");
+const { storeUserDeviceInfo } = require("../controllers/userDeviceInfo");
+
+const router = express.Router();
+
+router.post("/", storeUserDeviceInfo);
```

Comment:
add a validator middleware please, or else I can pass any other key which has a javascript that might lead to stored XSS.

Reply:
in progress

Reply:
Added validation for this @RitikJaiswal75 @shreya-mishra